components/places/sql/create_shared_triggers.sql

```diff
@@ -61,7 +61,12 @@ BEGIN
     WHERE NEW.guid = 'root________'
       AND NEW.parent IS NOT NULL;
 
-    SELECT throw('update: item without parent')
+    SELECT throw(format(
```

Comment:
So this probably won't help us _right now_ as we don't hit this constraint anymore since we've relaxed the trigger, however I think we should keep the stronger debugging here incase we decide to risk it and try change back to the more "correct" trigger

Reply:
I'm not sure about the new string - IIUC, it will mean each user with this problem will be reporting a different string (so counting it will be tricky?), and the specific unique string isn't going to help them or us? Even if we got a user who captured their exact string, how would it help us or that specific user?

(the constant operation does make sense though!)

Reply:
Thinking about this more, I'm inclined to agree -- I was thinking if there was a possibility that we'd see potentially some static guids like `root__..., menu____` but those would be captured by other triggers. I removed them for now to keep things simpler as we'll have better logging in the actual code.

Reply:
that would have kinda made sense, but I think they were logging `parent` which is the rowid.